vulture/core.py

```diff
@@ -457,26 +495,41 @@ def _define(
         confidence=DEFAULT_CONFIDENCE,
         ignore=None,
     ):
+        def ignored(name):
+            return (ignore and ignore(self.filename, name)) or _match(
+                name, self.ignore_names
+            )
+
+        def has_noqa(lineno, typ):
+            if typ == "property" and sys.version_info >= (3, 8):
+                # python3.8 onwards, lineno for property is the same as that of
+                # the decorated method.
+                lineno = lineno - 1
```

Comment:
Also, I think, it rather makes sense to ignore this case:

```py
class Foo:
    @property
    def bar():  # noqa
        pass
```

than adding `# noqa` infront of `@property`?

Reply:
I think adding the noqa on the same line as bar() is the better solution, yes. We should change this at a lower level as well: in all Python versions, we should report the line containing bar() as the line number for the property. This should be a separate PR though. However, this might be tricky in case there are multiple @property decorators. Could you try this out, please?